PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,96 @@
+### Description
+<!-- Add description of work done here -->
+
+### Spec
+
+Designs: [Designs](DESIGN_URL)
+
+See Story: [ISSUE_NUMBER](ISSUE_URL)
```

Comment:
We can make this a little easier by adding most of the link, then it's a matter of typing the issue number:

```suggestion
See Story: [ISSUE_NUMBER](https://sparkbox.atlassian.net/browse/FSA22V1-ISSUE_NUMBER)
```